Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html

```diff
@@ -28,10 +28,10 @@
 		</f:for>
 	</f:if>
 
-	<script type="text/javascript">
+	<script type="text/javascript"><![CDATA[
 		// TODO: Get rid of those global variables
 		{neos:backend.javascriptConfiguration()}
```

Comment:
Is this line still evaluated by fluid when wrapped in CDATA[] ?

Reply:
no, I don't think it is!

Reply:
So if this causes the problem, it needs to be fixed another way.

Reply:
Right, I added this "after the fact" to have all JS wrapped for consistency. Not needed and breaks the inclusion (even though the module still loads.)